csp-mixin/signals/gcpceOverview.libsonnet

```
@@ -0,0 +1,300 @@
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+            exprWrappers: [['100 *', '']],
+          },
+        },
+      },
+      tableUptime: {
+        name: 'tableUptime',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'stackdriver_gce_instance_compute_googleapis_com_instance_uptime_total{%(queriesSelector)s}',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableSentPackets: {
+        name: 'tableSentPackets',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_sent_packets_count{%(queriesSelector)s}[5m])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableReceivedPackets: {
+        name: 'tableReceivedPackets',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_received_packets_count{%(queriesSelector)s}[5m])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableSentBytes: {
+        name: 'tableSentBytes',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_sent_bytes_count{%(queriesSelector)s}[5m])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableReceivedBytes: {
+        name: 'tableReceivedBytes',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_received_bytes_count{%(queriesSelector)s}[5m])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableReadBytes: {
+        name: 'tableReadBytes',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'rate(stackdriver_gce_instance_compute_googleapis_com_instance_disk_read_bytes_count{%(queriesSelector)s}[5m])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      tableWriteBytes: {
+        name: 'tableWriteBytes',
+        type: 'gauge',
+        sources: {
+          stackdriver: {
+            expr: 'rate(stackdriver_gce_instance_compute_googleapis_com_instance_disk_write_bytes_count{%(queriesSelector)s}[5m])',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+          },
+        },
+      },
+      // end table
+
+      memoryUtilization: {
+        name: 'Memory Utilization',
+        description: 'Memory Utilization',
```

Can we use always the same pattern? First word in uppercase, the rest in lowercase?
```suggestion
        description: 'Memory utilization',
```

---

I don't see a strong rule about this, when I look at other dashbaords.
I think I prefer having Capital letters for each word except for prepositions

---

in my PRs I have comments about having standard naming on the titles for that reason I think we can use the same. If people in the past didn't do it does not mean we can decide to start using it